code/modules/jobs/job_types/roboticist.dm

```diff
@@ -23,6 +23,7 @@
 
 	display_order = JOB_DISPLAY_ORDER_ROBOTICIST
 	departments = DEPARTMENT_SCIENCE
+	rpg_title = "Necromancer"
```

Comment:
I think Golemancer fits more. Give Necromancer to Paramedic.

Reply:
Agreed